README.md

```diff
@@ -90,9 +96,15 @@ To install these userstyles:
     - Replacing `<name>` with the name of the userstyle.
 4. Enjoy!
 
-### Contributing
+> **Note** <br>
+> If you are using Oprea please see [oprea.md](docs\oprea.md) for instructions as it differs from the normal process.
```

Comment:
Opera stylus is deprecated.

Reply:
Opera stylus maybe deprecated. However, since the engine is now using chromium it can run therefore I just wanted to add this in since it was a little niche.

Reply:
This should be renamed to Opera / `opera.md`, if you're going to keep this.